README.md

```diff
@@ -1,8 +1,52 @@
 # Project Guide-Doge
 
-Data visualization experiments
+> #### Table of Contents
+> - [Problem](#problem)
+> - [Experiments](#experiments)
+>   - [Audification](#audification)
+>   - Data Table
+>   - Text Summary of Key Points
+> - [References](#references)
+> - [Contributing](#contributing)
 
-## Prerequisites
+## Problem
+WHO (World Health Organization) estimates that at least 2.2 billion people have a visual impairment or blindness. While HTML5 comes with various features to improve screen-reader accessibility including accessible data tables, text alternatives, skip links, etc., most of the data visualization are still inaccessible by visually impaired or blind people.
+
+## Experiments
+In an attempt to improve accessibility to data visualization, a number of experiments will be done in this repo to find alternatives to it.
```

Comment:
alternatives to what?

Reply:
I meant alternatives to data visualization, but probably I can reword it better.

Reply:
How about something like this: `In an attempt to improve accessibility to data visualization, a number of experiments will be done in this repo to explore improvements to data consumption for impaired users.`

Reply:
Sounds clearer :+1: